boards/shields/mikroe_wifi_bt_click/doc/index.rst

```diff
@@ -0,0 +1,78 @@
+.. _mikroe_wifi_bt_click:
+
+MikroElektronika WIFI and BLE Shield
+####################################
+
+Overview
+********
+
+This shield contains an Espressif Systems ESP32 which is responsible
+to offload the WIFI and Bluetooth tasks from the host processor. It uses
+the standard MikroBus interface with UART pins plus its Espressif AT
+command set. Alternatively, the Zephyr modem driver can be used,
+configuring the module as a native WIFI networking interface.
+
+.. image:: ./wifi_click.jpeg
+   :width: 634px
+   :align: center
+   :alt: MikroE WIFI BLE Click
+
+Pins Assignment of the Shield
+=============================
+
++-----------------------+---------------------------------------------+
+| Shield Connector Pin  | Function                                    |
++=======================+=============================================+
+| RST#                  | Reset pin, in most of the boards it should  |
+|                       | be disconnected because its tied to GND     |
++-----------------------+---------------------------------------------+
+| TX                    | UART TX communication pin                   |
++-----------------------+---------------------------------------------+
+| RX                    | UART RX communication pin                   |
++-----------------------+---------------------------------------------+
+
+Requirements
+************
+
+This shield requires a board which provides a configuration that allows an
+UART interface. (see :ref:`shields` for more details).
+
+.. note::
+   Sometimes boards declare standard headers like Arduino R3 but don't
+   define all connections.  Make sure that the board you are using includes
+   all required definitions to avoid build errors.
+
+This shield should be loaded with the `ESP32 AT Bin`_ software which is
+available at Espressif Systems web site. This version is command
+compatible with ESP32 AT Bin 2.0.0
+
+.. note::
+   Boards that already have a network interface:  Check the network
+   documentation to understand how to properly configure both interfaces.
+   To keep things simple, make sure WiFi is the only interface enabled in
+   Networking -> Link Layer Options.  This will avoid problems running
+   Zephyr samples.
+
+Build and Programming
+*********************
+
+Set ``-DSHIELD=<mikroe_wiffi_bt>`` when you invoke ``west build``.
```

Comment:
```suggestion
Set ``-DSHIELD=<mikroe_wifi_bt>`` when you invoke ``west build``.
```

Reply:
Spelling (`wiffi`), but is the shield name `mikroe_wifi_bt` or `mikroe_wifi_bt_click`?